html/semantics/embedded-content/the-iframe-element/iframe-loading-lazy-in-script-disabled-iframe.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<head>
+<title>Iframes with loading='lazy' in script disabled iframe are not handled
+       as 'lazy'</title>
+<link rel="help" href="https://github.com/scott-little/lazyload">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+</head>
+
+<div style="height:1000vh;"></div>
+<iframe id="iframe" sandbox="allow-same-origin"
+        src="resources/iframe-loading-lazy-in-viewport.html">
+</iframe>
+<script>
+promise_test(async t => {
+  await new Promise(resolve => iframe.addEventListener("load", resolve));
+
+  const inner_iframe = iframe.contentDocument.querySelector("iframe");
+
+  assert_true(inner_iframe.contentDocument != null,
```

Comment:
Why would this be null though? Isn't there still the initial about:blank, whether lazy-loaded or not?

Reply:
Maybe this can instead do something like 
```js
assert_equals(inner_iframe.contentDocument.body.textContent.trim(), 'Subframe');
```

Reply:
Right, it may have worked for WebKit, anyway Simon's suggestion is better and I verified that it works in WebKit if my patch is applied (https://bugs.webkit.org/show_bug.cgi?id=217315).